.travis.yml

```diff
@@ -72,12 +72,12 @@ script:
 
 after_script:
   # Killing via SIGTERM in order to trigger atexit and dump coverage information in WSGI
-  - "docker-compose -f docker-compose.test.yml kill -s SIGTERM"
+  - "docker-compose -f docker-compose.test.yml kill -s SIGHUP web"
+  - "docker-compose -f docker-compose.test.yml kill"
   - "docker-compose -f docker-compose.test.yml rm -f"
 
 after_success:
-  - mv .coverage .coverage.tests
-  - coverage combine
   # Coverage report contains Docker paths. We replace them, so that we can run Coveralls.
   - sed -i 's@\"/code/@'"\"$(pwd)/"'@g' .coverage
+  - cat .coverage
```

Comment:
Debugging. It's part of the last WIP commit. Yesterday I tried to really have coverage on Selenium tests, but couldn't make it work.